components/common/PanelsCreator.tsx

```diff
@@ -214,6 +214,16 @@ function displayTheCorrectListItem(
           index={index}
         />
       );
+    case "otherSpecialties":
+      return panel[panelProp]?.length > 0
+        ? panel[panelProp].map(
+            ({ specialtyId, name }: SpecialtyType): JSX.Element => (
+              <div key={specialtyId}>
+                <div data-cy={`otherSpecialty${specialtyId}Val`}>{name}</div>
+              </div>
+            )
+          )
+        : "None provided";
```

Comment:
One thing I was wondering: less than 1% of placements have 'Other specialties' - is it worth using up screen real estate to show 'None provided'? Or is it just easier to keep it consistent with how all of the placement details are displayed?

Reply:
Yeah, bit of a judgement call this one. Real estate-wise makes little difference and as you say, it’s consistent. Also showing it might trigger those users who expect to see something there etc.